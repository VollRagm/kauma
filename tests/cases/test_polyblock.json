{
    "testcases": {
        "b856d760-023d-4b00-bad2-15d2b6da22fe": {
            "action": "poly2block",
            "arguments": {
                "semantic": "xex",
                "coefficients": [
                    12,
                    127,
                    9,
                    0
                ]
            }
        },
        "254eaee7-05fd-4e0d-8292-9b658a852245": {
            "action": "block2poly",
            "arguments": {
                "semantic": "xex",
                "block": "ARIAAAAAAAAAAAAAAAAAgA=="
            }
        }
    }
}