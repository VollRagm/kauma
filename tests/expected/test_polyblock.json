{
    "responses": {
        "b856d760-023d-4b00-bad2-15d2b6da22fe": {
            "block": "ARIAAAAAAAAAAAAAAAAAgA=="
        },
        "254eaee7-05fd-4e0d-8292-9b658a852245": {
            "coefficients": [
                0,
                9,
                12,
                127
            ]
        }
    }
}